nimare/generate.py

```diff
@@ -105,6 +111,61 @@ def create_coordinate_dataset(
     return ground_truth_foci, dataset
 
 
+def create_neurovault_dataset(
+    collection_ids=NEUROVAULT_IDS,
+    contrasts=CONTRAST_OF_INTEREST,
+    img_dir=None,
+    map_type_conversion=None,
+    **dset_kwargs,
+):
+    """Download images from NeuroVault and use them to create a dataset.
+
+    Parameters
```

Comment:
```suggestion

    This function will also attempt to generate Z images for any contrasts
    for which this is possible.

    Parameters
```

Maybe something like this?

Reply:
oh, oops, sorry, for the function `create_neurovault_dataset` (not `convert_neurovault_to_dataset`) that makes sense, yes.